apps/auction/routes.coffee

```diff
@@ -29,10 +29,9 @@ setupUser = (user, auction) ->
           user.set 'registered_to_bid', boolean
         error: ->
           user.set 'registered_to_bid', false
+      ).then( ([ bidder ]) ->
+        user.set 'qualified_for_bidding', bidder?.qualified_for_bidding
```

Comment:
Aha, it's intentional that these execute in parallel, since `checkRegisteredForAuction` just depends on the access token and `user.fetch()` is for grabbing additional info on the user. Also since `Q.all` is accepting an array here, putting them on separate lines is the same as adding a comma, but let me know if I'm missing something!